app/src/main/res/values/strings.xml

```diff
@@ -75,20 +75,20 @@
     <string name="seek_duration_title">Fast-forward/-rewind seek duration</string>
     <string name="download_thumbnail_title">Load thumbnails</string>
     <string name="show_comments_title">Show comments</string>
-    <string name="show_comments_summary">Disable to stop showing comments</string>
+    <string name="show_comments_summary">Turn off to hide comments</string>
     <string name="download_thumbnail_summary">Turn off to prevent loading thumbnails, saving data and memory usage. Changes clear both in-memory and on-disk image cache.</string>
     <string name="thumbnail_cache_wipe_complete_notice">Image cache wiped</string>
     <string name="metadata_cache_wipe_title">Wipe cached metadata</string>
     <string name="metadata_cache_wipe_summary">Remove all cached webpage data</string>
     <string name="metadata_cache_wipe_complete_notice">Metadata cache wiped</string>
     <string name="auto_queue_title">Auto-queue next stream</string>
-    <string name="auto_queue_summary">Auto-append a related stream when playing the last stream in a non-repeating queue</string>
+    <string name="auto_queue_summary">Auto-append a related stream (to non-repeating queue) after last file</string>
```

Comment:
I suggest stream because we do not play files. Or did you replace stream to have no repetition?
```suggestion
    <string name="auto_queue_summary">Auto-append a related stream (to non-repeating queue) after last stream</string>
```